src/main/java/edu/kis/powp/jobs2d/command/ICompoundCommand.java

```diff
@@ -5,8 +5,5 @@
 /**
  * Interface extending Job2dDriverCommand to execute more than one command.
  */
-public interface ICompoundCommand extends DriverCommand {
-
-    public Iterator<DriverCommand> iterator();
-
+public interface ICompoundCommand extends DriverCommand, Iterable<DriverCommand> {
```

Comment:
thanks for the small refactoring, but it would be ideal if it was a separate PR.